src/style.css

```diff
@@ -1,3 +1,58 @@
+:root {
+  --color-primary: #00acdc; /* Define a variable for primary color */
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a generic font family as a fallback after 'Arial', such as 'sans-serif'.